README.md

```diff
@@ -6,6 +6,7 @@
 
 - HTML 
 - CSS
+- javaScript
 
 ## Project link
 
```

Comment:
- [ ]  Your project documentation should contain all contributors to the project. Please update your readme file and add your partner as a contributor.

Reply:
- [ ] As suggested previously, kindly add your collaborator details below Author 1 and add all contact details of your collaborator.